app/src/main/java/com/plan2meet/dto/CalendarDTO.kt

```diff
@@ -7,10 +7,11 @@ class CalendarDTO (
     var year        : Year,
     var month       : Month,
     var dates       : Array<DateDTO>,
-    var groupDTO    : GroupDTO
+    var groupDTO    : GroupDTO,
         )
 {
+    //Returns true if this CalendarDTO object is equal to the calendarDTO object stored in the groupDTO object's member variable calendarDTO.
```

Comment:
Above classes and functions, use proper KDoc.
https://kotlinlang.org/docs/kotlin-doc.html#module-and-package-documentation